netam/molevol.py

```diff
@@ -320,7 +324,7 @@ def build_codon_mutsel(
     codon_probs = codon_probs_of_mutation_matrices(mut_matrices)
 
     if multihit_model is not None:
-        codon_probs = multihit_model(parent_codon_idxs, codon_probs)
+        codon_probs = multihit_model.forward(parent_codon_idxs, codon_probs)
```

Comment:
I would have thought these were identical.

Reply:
They are, but I like the style of an explicit named method call for code searching purposes